tasks/lzards-backup/src/index.ts

```diff
@@ -82,12 +83,13 @@ export const generateDirectS3Url = async (params: {
         secretAccessKey,
         accessKeyId,
         sessionToken,
+        expiration,
       },
     };
   }
   const s3ObjectStore = new S3ObjectStore(s3Config);
   const s3Uri = buildS3Uri(Bucket, Key);
-  return await s3ObjectStore.signGetObject(s3Uri, {}, { Expires: s3AccessTimeoutSeconds });
+  return await s3ObjectStore.signGetObject(s3Uri, {}, {}, { expiresIn: s3AccessTimeoutSeconds });
```

Comment:
`expiresIn` should be passed as `presignOptions`, not as a query parameter

Reply:
Is it not possible to make a unit test to test this?

Reply:
localstack doesn't detect this type of SignatureDoesNotMatch error unfortunately.